python/python/bystro/proteomics/annotation_interface.py

```diff
@@ -227,5 +229,15 @@ def get_samples_and_genes_from_query(
 ) -> pd.DataFrame:
     """Given a query and index, return a dataframe of (sample_id, gene, dosage) rows matching query."""
     query = _build_opensearch_query_from_query_string(user_query_string)
-    samples_and_genes_df = _run_annotation_query(query, index_name, client)
-    return samples_and_genes_df
+    return _run_annotation_query(query, index_name, client)
+
+
+def join_annotation_results_to_proteomics(
+    query_result_df: pd.DataFrame, tmt_dataset: TandemMassTagDataset
```

Comment:
Just to make sure I'm following correctly, the query_result_df going in here including the columns sample_id and gene names is the final result generated by _get_samples_genes_dosages_from_hit that gets kicked off by get_samples_and_genes_from_query?

Reply:
yes, maybe `get_samples_and_genes_from_query` should be renamed in order to make that clearer

Reply:
maybe just a docstring to indicate where query_result_df comes from to start following the thread? just a little extra clarification would be appreciated